### ISSUE TITLE
Fix: Duplicate js and tsx files

### ISSUE THREAD

**Issue by jorgemanzo** [peerist/capstone#15]
There are now both js and tsx files in master, particularly in the `pages/` directory. It seems like we're not using `.tsx` anymore, and having `app.js` and `app.tsx` is confusing next. We should find the differences between them and leave only `.js` files.

**Comment by jorgemanzo** [comment]
In  f849f16 , I looked through the differences between every duplicate in `pages/`. For the most part, it just seemed like we were trying to use Rebass to apply the styles in our `theme/` folder. Since that seemed to be the intention, I made our emotion style components use the css in `theme/`. Though, It seems like it could use some work.
 
![Screenshot from 2020-03-07 01-08-52](https://user-images.githubusercontent.com/26660036/76140517-3d280e80-6010-11ea-8a4a-b1a61246cbf2.png)

![Screenshot from 2020-03-07 01-07-37](https://user-images.githubusercontent.com/26660036/76140510-25e92100-6010-11ea-9817-0ca364cfd614.png)
